.github/workflows/build-and-publish.yml

```diff
@@ -95,24 +96,19 @@ jobs:
             **/gradle/**
             Scripts/**
   
-  githubRelease:
+  create_Tag:
     needs: maven_Release
     runs-on: ubuntu-latest
+    env:
+      RELEASE_TAG: ""
     steps:
       - uses: actions/checkout@v2
-      - name: Download Build Artifact
-        uses: actions/download-artifact@v2.0.10
-        with: 
-          name: drop
-          path: drop        
-      - name: Github Release
-        uses: anton-yurchenko/git-release@v4.1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
-          DRAFT_RELEASE: "false"
-          PRE_RELEASE: "false"
-          CHANGELOG_FILE: "CHANGELOG.md"
-          ALLOW_EMPTY_CHANGELOG: "true"
+      - name: Get Version
+        id: GetVersion
+        run: .\scripts\getLatestVersion.ps1
+        shell: pwsh
+      - name: Create tag
+        uses: rickstaa/action-create-tag@v1.2.0
         with:
-          args: |
-              drop/**
+          tag: ${{ steps.GetVersion.outputs.tag }}
```

Comment:
we will probably need to queue the workflow explicitly because of the whole "a bot cannot trigger a workflow" thing.